buckets/local/options.go

```diff
@@ -2,14 +2,14 @@ package local
 
 import (
 	cid "github.com/ipfs/go-cid"
-	"github.com/textileio/go-threads/core/thread"
 )
 
 type newOptions struct {
 	name    string
 	private bool
-	thread  thread.ID
-	key     string
+	// (jsign): unused?
+	//thread  thread.ID
+	//key     string
```

Comment:
@sanderpick , got some lint about unused fields. 
What's your opinion to do here? (left this code now to remember to comment in the review)

Reply:
Cool, will delete then. 👍